src/extension.ts

```diff
@@ -208,8 +208,9 @@ class ClientWorkspace {
     };
 
     const pattern = this.config.multiProjectEnabled
-      ? `${this.folder.uri.path}/**`
+      ? `**${this.folder.uri.path}/**`
```

Comment:
So I'm still wondering about this change... What exactly is the output for both `.path` and `.fsPath` and why do we need to prefix that with a wildcard?

Is the wildcard supposed to make the pattern drive-letter-agnostic, e.g. it'd match both `C:\Some\Path` and `D:\Some\Path` with a `**/some/path/**` pattern?

Reply:
right that's a really good point.

for fsPath it's what you would expect just standard operative system paths
windows: `c:\Some\Path`
linux: `/some/path`

path is what is interesting (it's basically the path part of a FILE uri):
windows: `/c:/Some/Path` with the file uri: `file:///c:/Some/Path`
linux: `/Some/Path` with the file uri: `file:///Some/Path`

Now the pattern matcher uses file uri paths for matching, my guess is they wanted to have a OS agnostic pattern matcher.

The `**` is because windows puts something else infront of the files when matching and I have no clue what however the only way to get around it was `**` which will catch it.

Reply:
Maybe a better solution would be to only apply this for linux

Reply:
Thanks for clarifying! What do you mean by something else? Do you mean like `c%3A` (which is percent-encoded `c:`)?

Reply:
No, I meant they put something `[something]/c:/...` I don't know what that something is and I couldn't figure it out, I thought it was fine however but I understand wanting to fix it the correct way :).